src/main/resources/view/DarkTheme.css

```diff
@@ -1,32 +1,32 @@
 .background {
-    -fx-background-color: derive(#1d1d1d, 20%);
-    background-color: #383838; /* Used in the default.html file */
+    -fx-background-color: derive(#ffffff, 20%);
+    background-color: #ffffff; /* Used in the default.html file */
```

Comment:
This doesn't cause issues now but the ScrollPane component in JavaFX will make this cause some issues, and ScrollPane is needed to show contact details. I think we should wait on this adjustment until we finish 1.2 and we can commit fully to changing the UI.